04-onion-routing.md

```diff
@@ -155,52 +151,50 @@ The overall structure of the packet is as follows:
 2. data:
    * [`1`:`version`]
    * [`33`:`public_key`]
-   * [`20*65`:`hops_data`]
+   * [`1300`:`hops_data`]
    * [`32`:`hmac`]
 
 For this specification (_version 0_), `version` has a constant value of `0x00`.
 
-The `hops_data` field is a structure that holds obfuscations of the next hop's
-address, transfer information, and its associated HMAC. It is 1300 bytes (`20x65`) long
-and has the following structure:
+The `hop_payloads` field is a structure that holds obfuscated routing information, and associated HMAC.
```

Comment:
nit: this is still named `hops_data` on line 154. I'm fine with both names but we should use the same everywhere :)

Reply:
No, the difference really is between `hop_payloads` which is a collection of either `hop_data` or `hop_payload` values, pluralization matters in this case :-)